source/agora/node/RemoteNode.d

```diff
@@ -170,4 +170,25 @@ class RemoteNode
             logError(ex.message);
         }
     }
+
+    /***************************************************************************
+
+        Returns:
+            the height of the node's ledger,
+            or 0 if the request failed
+
+    ***************************************************************************/
+
+    public ulong getBlockHeight ( )
+    {
+        try
+        {
+            return this.api.getBlockHeight();
+        }
+        catch (Exception ex)
+        {
+            logError(ex.message);
+            return 0;
```

Comment:
`ulong.max`

Reply:
Good point, block height is zero for genesis blocks. I'll add a test-case. Also like I said before I dislike this class, but I plan on making it better at a later point. There's no retry mechanism for example.

Reply:
Ping